functions/index.js

```diff
@@ -0,0 +1,208 @@
+// Require'd modules
+const functions = require("firebase-functions");
+const admin = require("firebase-admin");
+const axios = require("axios");
+const cheerio = require("cheerio");
+const cors = require("cors");
+const { Translate } = require("@google-cloud/translate");
+
+// require'd local modules
+const { proxyRequest } = require("./proxy");
+
+const corsHandler = cors({
+    // TODO: get deploy url
+    origin: ["localhost:3000"]
```

Comment:
corsHandler wraps all functions and allows us to whitelist frontend URL's for Firebase Functions use, (we'll need to update this when we actually deploy the front-end somewhere)